scripts/mkcloud

```diff
@@ -284,6 +284,10 @@ function cleanup()
         kpartx -dsv /dev/mapper/$vol
     done
 
+    # workaround host grabbing guest devices
+    for vol in postgresql rabbitmq ; do
+        dmsetup remove drbd-$vol
+    done
```

Comment:
no, I think that would not work as the above loop filters for $cloud
and is about partition mappings - not about VGs/LVs like the drbd one